docs/src/existing_mnemonic.md

```diff
@@ -0,0 +1,31 @@
+# existing-mnemonic
+
+## Description
+Uses an existing BIP-39 mnemonic phrase for key generation.
+
+## Optional Arguments
+
+- **`--chain`**: The chain to use for generating the deposit data. Options are: 'mainnet', 'holesky', etc...
+
+- **`--mnemonic`**: The mnemonic you used to create withdrawal credentials.
+
+- **`--mnemonic_password`**: The mnemonic password you used in your key generation. Note: It's not the keystore password.
+
+- **`--validator_start_index`**: The index of the first validator's keys you wish to generate. If this is your first time generating keys with this mnemonic, use 0. If you have generated keys using this mnemonic before, use the next index from which you want to start generating keys from (eg, if you've generated 4 keys before (keys #0, #1, #2, #3), then enter 4 here.
+
+- **`--num_validators`**: Number of validators to create.
+
+- **`--keystore_password`**: The password that is used to encrypt the provided keystore. Note: It's not your mnemonic password.
+
+- **`--execution_address`**: The Ethereum 1 address for validator withdrawals.
```

Comment:
We should avoid making references to Ethereum 1 (ETH 1) or Ethereum 2 (ETH 2). Related to #29 . It could be done in a separate pass to keep everything consistent or it could be done with this PR.

Reply:
Did a pass to remove ETH1 from messaging. There is still variable names all over the place but I think that is outside the scope.